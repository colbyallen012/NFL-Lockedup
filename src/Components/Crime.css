.crime {
  background-color: lightcyan;
  color: black;
  margin: 10px;
  padding: 10px;
  height: 25vh;
  width:  415px;
}