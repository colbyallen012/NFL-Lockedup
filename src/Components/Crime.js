import React from 'react';
import './Crime.css';

const Crime = ({category, arrests}) => {
  return (
    <div className='crime'>
      <h3>{category}</h3>
      <p>Arrests: {arrests}</p>
    </div>
  )
}

export default Crime;