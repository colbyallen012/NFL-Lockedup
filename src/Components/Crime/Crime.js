import React from 'react';
import PropTypes from 'prop-types';
import './Crime.css';

const Crime = ({category, arrests}) => {
  return (
    <div className='crime'>
      <h3 className='crime-crime'>{category}</h3>
      <p className='crime-arrests'>Arrest Count: {arrests}</p>
    </div>
  )
}

Crime.propTypes = {
  category: PropTypes.string.isRequired,
  arrests: PropTypes.string.isRequired
}

export default Crime;