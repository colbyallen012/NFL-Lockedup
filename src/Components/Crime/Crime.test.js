import React from 'react';
import { shallow } from 'enzyme';
import Crime from './Crime';

describe('Crime', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Crime category='DUI' arrests={20} />)
  })

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })
})