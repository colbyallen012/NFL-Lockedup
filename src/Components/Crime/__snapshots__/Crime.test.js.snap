// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Crime should match the snapshot 1`] = `
<div
  className="crime"
>
  <h3
    className="crime-crime"
  >
    DUI
  </h3>
  <p
    className="crime-arrests"
  >
    Arrest Count: 
    20
  </p>
</div>
`;
